cltk/corpus/old_norse/alphabet.py

```diff
@@ -0,0 +1,18 @@
+#all the alphabets in old norse
+ALPHABET = ['a','b','c','d','ð','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z','þ','æ','ǫ','ø','œ']
+
+#All the vowels in old norse
+VOWELS = ['a', 'á', 'e', 'é', 'i', 'í', 'o', 'ó', 'u', 'ú', 'y', 'ý', 'æ','ǫ́', 'ǫ', 'ø', 'œ']
+
+#All the consonants in old norse
+CONSONANTS = ["b", "c", "d", "ð", "f", "g", "h", "j", "k", "l", "m", "n", "p", "q", "r", "s", "t", "v", "w", "x", "z", "þ"]
+
+#All the Dipthongs in old norse
+DIPTHONGS = ["au","ei","ey"]
+
+#Similar to other Germanic languages, the vowel change, or umlaut, operates in Old Norse. There are two kinds: i-umlaut, and u-umlaut.
+#The i-umlaut appears thus:
+UMLAUT_I = {"a": "e", "á": "æ", "o": "ø", "ó": "œ", "u": "y", "ú": "ý"}
```

Comment:
@aakash5897 Would you please help me understand the two dicts for umlauts? What is the purpose of them? Give an example of how someone would use it.